Simplified complete diagram update

francardoso93 opened the pull request:
- Converted from Miro to Draw.io (Making our life easier for next updates)
- Add notification topics
- Simplified and more meaningful database representation
- Dark Mode
- Rename to "Elastic IPFS"

alanshaw reviewed (commented):
I ❤️ this but I'm not sure the E-IPFS client and event bit is right...?

francardoso93 commented:
> I heart this but I'm not sure the E-IPFS client and event bit is right...?

The notification could come back directly to the client and also to other external systems, it really depends on who is interested on getting that. So far I know this will be used by W3 processor and maybe the websocket components we've been discussing. To avoid confusion, I've removed that final "notify" arrow.
Also added the zoomed in diagrams for the subystems

alanshaw reviewed (approved):
LGTM
